### DISCUSSION TITLE
IIS反代,Onedrive直链下载链接异常

### DISCUSSION THREAD

**Discussion by QXAzusa** [AlistGo/alist#897] [Q&A] [answered]
## 环境
IIS 8
Alist版本号: v2.3.2
## 问题
在服务器(localhost:5244)访问,所给的下载链接是正确的
反代后在服务器上访问,获取的下载链接不正确
![图片](https://user-images.githubusercontent.com/69675439/161431104-b236d269-a59b-43dd-af2b-68d2c6202d9f.png)
API返回的链接正常
![图片](https://user-images.githubusercontent.com/69675439/161431478-351f9599-5f71-4c0e-897e-456b171ee2f1.png)
### 反代规则
![图片](https://user-images.githubusercontent.com/69675439/161431708-d82753d2-32fd-4789-bad5-09a6eab1ebef.png)

**Comment by xhofe** [comment, marked as answer]
这种iis的问题应该问iis啊。。

百度/Google搜 `iis reverse rewrite host in response headers`

**Reply by QXAzusa** [reply]
抱歉，已经用重写规则解决了，有点考虑不周
